### PR TITLE
V2.11.0 rc1 fixes

### PR DESCRIPTION
**Pull request by Salanto**
Fixes some minor issues in 2.11 RC1

### CODE REVIEW

File: src/widgets/moderator_dialog.cpp [hunk 1]
```diff
@@ -11,6 +11,9 @@
 #include <QVBoxLayout>
 
 const QString ModeratorDialog::UI_FILE_PATH = "moderator_action_dialog.ui";
+const int minute = 60;
+const int hour = 3600;
+const int day = 86400;
```
Comment (stonedDiscord) [inline src/widgets/moderator_dialog.cpp:16]: surely, qt already has a solution for this

File: src/widgets/moderator_dialog.cpp [hunk 2]
```diff
@@ -87,7 +94,8 @@ void ModeratorDialog::onAcceptedClicked()
     }
     else
     {
-      arglist.append(QString::number(ui_duration->value()));
+      qint64 unix_duration = ui_duration_mm->value() * minute + ui_duration_hh->value() * hour + ui_duration_dd->value() * day;
```
Comment (stonedDiscord) [inline src/widgets/moderator_dialog.cpp:97]: literally unreadable

**Review by stonedDiscord — CHANGES_REQUESTED** [review]

**Comment by Salanto** [comment]
This looks even worse, ngl, u sure I can't just use const int to make the conversions and cleanup the addition part a bit?

**Comment by stonedDiscord** [comment]
I think this is acutally better, the oneliner was very dense.
Now it's clear how the values from the box get converted to what the box says, then to seconds and finally get added up.

**Review by stonedDiscord — APPROVED** [review]

**Comment by Salanto** [comment]
Works for me

**Pull request merged**